backend/src/services/notification/deleteWeekly.js

```diff
@@ -0,0 +1,17 @@
+import NotificationRepo from "../../database/repositories/NotificationRepo.js";
+import { NotFoundError } from "../../lib/errors/index.js";
+import cron from "node-cron";
+
+export default async function deleteWeekly() {
+    let notifications = [];
+    
+    notifications = await NotificationRepo.deleteWeeklyNotifications();
+   
+    if (!notifications) throw new NotFoundError("An error occured while fetching notifications");
```

Comment:
There are other possible things that can cause the query to fail. I think serviceError would be more appropriate.

Reply:
Okay. Thank you, I will fix that.